presto-plugin-toolkit/src/main/java/com/facebook/presto/plugin/base/security/AllowAllAccessControl.java

```diff
@@ -25,6 +25,21 @@
 public class AllowAllAccessControl
         implements ConnectorAccessControl
 {
+    @Override
```

Comment:
can you add a test to prevent such situation in future

Reply:
Shall we generalize `TestPartitionsAwareAccessControl` to other connectors?

Reply:
I'll do that as a follow on.  This was just a simple opportunistic fix, and I don't want to get caught up with it in this PR.

Reply:
What about extracting `Add create/delete/rename schema in the connector AllowAllAccessControl` to a separate PR then?